docs/setup/rke2-cluster.md

```diff
@@ -0,0 +1,67 @@
+# Intro 
```

Comment:
I think this commit should be part of #27 - perhaps rebase to drop this commit from the branch for this PR?

Reply:
That worked, thank you! Just copied the commands and rebased the branch.